test/e2e/filter_test.go

```diff
@@ -62,12 +62,13 @@ data:
 			RunOpts{IntoNs: true, StdinReader: strings.NewReader(yaml1)})
 		expectedOutput1 := `
 Namespace  Name           Kind       Age  Op      Op st.  Wait to    Rs  Ri  
```

Comment:
Do you think something like this would be more readable?

```go
		expectedOutput1 := fmt.Sprintf(`
Namespace  Name           Kind       Age  Op      Op st.  Wait to    Rs  Ri  
%s  redis-config   ConfigMap  -    create  -       reconcile  -   -`, env.Namespace)
```

Reply:
Same for other similar places.